TryingMethods.java

```diff
@@ -1,43 +1,30 @@
-// This is the class TryingMethods
-
-// Create a class with the name "TryingMethods"
-// Add comments to describe what this class does
-
-// Add the a main method to this class
-
-// Add at least four methods in this class
-
-// Two of the method should be dynamic and the other two static
-
-// One of the dynamic methods should have an integer as an argument.
-// It return a string of the day of the week based on the argument that
-// was provided. Use a switch case in the body of the method.
-// The second dynamic method should return nothing but just print a
-// message to the console using an if statement with at least 3 blocks.
-// The conditions of the statement should be based on the argument or
-// parameter(s) of the method.
-
-// In one of the static methods use a while loop to find the sum of all
-// numbers less than 1000 and return the result.
+// Creating the TryingMethods class
+public class TryingMethods {
 
-// Lastly, in the last method use a for loop to find the sum of even
-// numbers between 12 and 103.
+    // Main Method
+    public static void main(String[] args){
 
-// ...
+        // Calling dynamic method 1
+        TryingMethods newObject = new TryingMethods();
+        System.out.println(newObject.firstChild(5));
 
-// Push your codes to github with at least 5 commits. Ensure to use at
-// least 5 branches to manage your code.
+        // calling dynamic method 2
+        TryingMethods secondObject = new TryingMethods();
+        secondObject.gradingSystem(2);
 
-// Find someone, who you will collaborate with. The collaborator will be
-// the one to accept the PRs you raise.
+        // Calling static method 2
+        staticSecondChild();
 
-// Thank you and happy coding.
+        // Calling static method 1
+        System.out.println(addingTwo());
 
-public class TryingMethods {
+    }
 
     // Dynamic Method 1
     public String firstChild(int day){
         String nameOfDay = "";
+
+        // switch case
```

Comment:
any reason for this comment?

Reply:
Just telling the peps that it's a switch case block, dah!